### PR TITLE
Fix military upkeep tooltip

### PR DESCRIPTION
**Pull request by lmoureaux**
There was probably a call to city_refresh_from_main_map when it was written (or I was using a governor).

See #1873.

### CODE REVIEW

File: client/text.cpp [hunk 1]
```diff
@@ -1884,30 +1884,33 @@ QString text_happiness_units(const struct city *pcity)
...
-    str += QString(PL_("resulting in <b>%1 additional unhappy citizen.</b>",
-                       "resulting in <b>%1 additional unhappy citizens.</b>",
-                       pcity->unit_happy_upkeep))
-               .arg(pcity->unit_happy_upkeep);
+    // TRANS: "This city supports X agressive military units, resulting in up
+    //        to %1 unhappy citizens." (second part)
+    str +=
+        QString(
+            PL_("resulting in up to <b>%1 additional unhappy citizen.</b>",
+                "resulting in up to <b>%1 additional unhappy citizens.</b>",
```
Comment (jwrober) [inline client/text.cpp:1911]: I think the `</b>` should be before the period. You don't need to bold the period. Otherwise rest looks great to me.

**Review by jwrober — APPROVED** [review]

File: client/text.cpp (hunk 1 above)
Comment (lmoureaux) [reply client/text.cpp:1911]: This would be [against typographic rules](https://writing.stackexchange.com/a/14161).

**Comment by lmoureaux** [comment]
I suppose this should be backported

File: client/text.cpp (hunk 1 above)
Comment (jwrober) [reply client/text.cpp:1911]: Interesting.

**Pull request merged**